### PR TITLE
Minor fixes on docs

### PR DESCRIPTION
**Pull request by viniciusccarvalho**

### CODE REVIEW

File: src/main/asciidoc/streaming/jdbc/http-mysql/pcf.adoc [hunk 1]
```diff
@@ -39,13 +39,13 @@ dataflow:>app import --uri {app-import-rabbit-maven}
 dataflow:>stream create --name mysqlstream --definition "http | jdbc --tableName=names --columns=name"
 Created new stream 'mysqlstream'
 
-dataflow:>stream deploy --name mysqlstream --properties "app.jdbc.spring.cloud.deployer.cloudfoundry.services=mysql"
+dataflow:>stream deploy --name mysqlstream --properties "deployer.jdbc.cloudfoundry.services=mysql"
```
Comment (ilayaperumalg) [inline src/main/asciidoc/streaming/jdbc/http-mysql/pcf.adoc]: Isn't this still a valid application property and hence it needs to have `app.` as the prefix? Same is the case for other places as well?

Comment (viniciusccarvalho) [reply src/main/asciidoc/streaming/jdbc/http-mysql/pcf.adoc]: Nope, it doesn't work on the way it was written in the docs. And according to the latest docs on the cf deployer. `deployer.<app>.cloudfoundry.services` is the correct way.

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
Rebased and merged as 92cd5f60370eabb4ae2d95884a33ee3734454212

**Pull request closed without merging**